core-web/libs/block-editor/src/lib/elements/dot-bubble-menu/dot-bubble-menu.component.html

```diff
@@ -177,5 +181,16 @@
     </button>
 </ng-template>
 
+<ng-template #bubbleContentMenu>
```

Comment:
Now these templates are no necessary with the new syntax, in this case this element could be inside if scope.